### PR TITLE
Merge v2 changes as of 08/02/2021

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: test/integration/run-test-v2.sh [hunk 1]
```diff
@@ -66,7 +66,15 @@ helm install azuredisk-csi-driver test/latest/azuredisk-csi-driver -n kube-syste
   --set image.azuredisk.tag=$image \
   --set azuredisk.cloudConfig="$(cat "${AZURE_CREDENTIAL_FILE}" | base64 | awk '{printf $0}'; echo)" \
   --set controller.port="${CONTROLLER_PORT}" \
+  --set azuredisk.nodeId="${nodeid_0}"
```
Comment (edreed) [inline test/integration/run-test-v2.sh:69]: Unfortunately, this line accidentally omitted the trailing backslash which results in the Azure credential file contents being written to the log when Helm dumps the chart output. I just now submitted a fix for this to the Mandalay master branch. Please pull that change into this PR.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**